framework/BaseForm.js

```diff
@@ -11,10 +11,10 @@ class BaseForm {
 
     async isFormDisplayed() {
         let isFormDisplayed;
-        let labelElement = await new Label(this.locator, this.name);
+        let labelElement = new Label(this.locator, this.name);
         try {
             isFormDisplayed = await labelElement.isDisplayed();
-        } catch (error) {
+        } catch{
```

Comment:
some older versions of JS may not recognise this syntax. I would recommend adding (e) or something like that to a catch block  for just in case